Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "semantic-release-cargo"
 version = "2.2.3"
-authors = ["Steven Bosnick <sbosnick@sympatico.ca>", "Eric Crosson <eric.s.crosson@utexas.edu>"]
+authors = [
```

Comment:
Apologies for the auto-formatting. Let me know if you'd like me to walk this back.

Reply:
All good!